README.md

```diff
@@ -6,147 +6,147 @@ Prestashop 1.6.x and 1.7.x
 ## Changelog
```

Comment:
Could you format this following https://keepachangelog.com/en/1.0.0/ and put it in CHANGELOG.md